src/components/common/SecureRoute.js

```diff
@@ -12,9 +12,12 @@ import { connect } from 'react-redux';
  * we reroute the user back to the profile selection page.
  */
 const ReduxSecureRoute = props => {
+    const { oktaAuth } = useOktaAuth();
+
+  const authService = oktaAuth;
   const { pathname } = useLocation();
   // If parent OR child isn't logged in, redirect to profile modal
```

Comment:
I changed the check here to just check if the user was logged in - this allows the user to access secure pathways even after they have clicked change user. This change is instead of classifying pages like Help as un-secure and allows any other page to be built either secure or un-secure yet still accessible. Pages that shouldn't be accessed until after the user has been selected are not shown via a ternary operator in the nav component.

Reply:
Sounds good.